README.md

```diff
@@ -105,6 +107,13 @@ designed for use in Google's codebase. The `FluentLogger` API is recommended for
 non-Google code, since its API should remain more stable over time.
 
 <a name="more-information"></a>
+
+## Using a different logging interface
+
+In addition to the `flogger-systemd-backend`, pull in an other backend, such as `flogger-log4j[2]-backend` or `flogger-slf4j-backend`, and set `flogger.backend_factory` system property according to the [instructions of the backend](https://github.com/google/flogger/blob/master/log4j/src/main/java/com/google/common/flogger/backend/log4j/Log4jBackendFactory.java#L30).
+
+To use `slf4j`, another option is using the third-party `jul-to-slf4j` library. Note that `flogger-slf4j-backend` and `jul-to-slf4j` differ in [how they map](https://github.com/google/flogger/issues/69#issuecomment-723043052) the `FINER` level.
```

Comment:
Thanks for pointing out the "FINER" thing, I'd be happy to see that unified if you think there's a good way to do it without breaking anyone.

Reply:
I don't think this would be a problem in practice, or at least didn't think it through enough. Calling it out like this is probably good enough.

Reply:
Cool.